docs/single-spa-api.md

```diff
@@ -49,8 +43,31 @@ or `null` (when the app doesn't exist). The string status is one of the followin
 - `SKIP_BECAUSE_BROKEN`: the app threw an error during load, bootstrap, mount, or unmount and has been
    siloed because it is misbehaving. Other apps may continue on normally, but this one will be skipped.
 
+## routing event
+single-spa fires an event `single-spa:routing-event` on the window every time that a routing event has occurred in which
+single-spa verified that all apps were correctly loaded, bootstrapped, mounted, and unmounted.
+This event will get fired after each hashchange, popstate, or triggerAppChange, even if no changes
+to child applications were necessary. Sample usage of this event might look like this:
+```js
+window.addEventListener('single-spa:routing-event', () => {
```

Comment:
Yeah, there is no extra data attached to the vanilla DOM event. You can call `singleSpa.getMountedApps()` or `singleSpa.getAppStatus()` if you want to find out what happened.